### ISSUE TITLE
Handle case where there is no .m2m file

### ISSUE THREAD

**Issue by candleshine** [sakuraapi/merge-to-master#7]
if I have no .m2m file I get the following error
```
$ npm i -g @sakuraapi/merge-to-master
/Users/daniel/.nvm/versions/node/v8.8.1/bin/m2m -> /Users/daniel/.nvm/versions/node/v8.8.1/lib/node_modules/@sakuraapi/merge-to-master/dist/index.js

> @sakuraapi/merge-to-master@0.3.0 install /Users/daniel/.nvm/versions/node/v8.8.1/lib/node_modules/@sakuraapi/merge-to-master
> chmod +x ./scripts/*.* ; true

chmod: ./scripts/*.*: No such file or directory
+ @sakuraapi/merge-to-master@0.3.0
updated 1 package in 2.538s

daniel~/Documents/Git Projects/ILB/ilb-portal on  develop
$ m2m -V
0.3.0
error: pathspec 'undefined' did not match any file(s) known to git.
unable to checkout undefined

daniel~/Documents/Git Projects/ILB/ilb-portal on  develop
$
```

**Comment by etsuo** [comment]
turns out this is actually related to a problem with the cleanup event handler... sorry about that.

**Comment by etsuo** [comment]
fixed on 0.3.1
